api/controllers/DashboardController.js

```diff
@@ -108,7 +107,6 @@ var dashboardController = {
             })
             .catch((error) => {
                 sails.log.error(error);
-                res.serverError();
```

Comment:
I assume sails will still handle rendering the error page?

Reply:
Ah good point, I honestly don't know why I removed that line in the first place